From 6d39013f83f3761770056064bd671586cb8802de Mon Sep 17 00:00:00 2001
From: Jean-Christophe PLAGNIOL-VILLARD <plagnioj@jcrosoft.com>
Date: Tue, 29 Jan 2013 22:10:04 +0100
Subject: [PATCH 305/397] ARM: at91: fix sama5d3 pmc reg

we more registre than 0x100 the last one is PMC_OCR at offset 0x110

Signed-off-by: Jean-Christophe PLAGNIOL-VILLARD <plagnioj@jcrosoft.com>
---
 arch/arm/boot/dts/sama5d3.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/sama5d3.dtsi b/arch/arm/boot/dts/sama5d3.dtsi
index f77c9a4..f175cf2 100644
--- a/arch/arm/boot/dts/sama5d3.dtsi
+++ b/arch/arm/boot/dts/sama5d3.dtsi
@@ -939,7 +939,7 @@
 
 			pmc: pmc@fffffc00 {
 				compatible = "atmel,at91rm9200-pmc";
-				reg = <0xfffffc00 0x100>;
+				reg = <0xfffffc00 0x120>;
 			};
 
 			rstc@fffffe00 {
-- 
1.8.5.3

